src/Card/CardContext.jsx

```diff
@@ -20,12 +23,15 @@ CardContextProvider.propTypes = {
   isLoading: PropTypes.bool,
   /** Specifies content of the component. */
   children: PropTypes.node,
+  /** Specifies `Card` style variant */
+  variant: PropTypes.oneOf(['light', 'dark', 'muted']),
```

Comment:
nit: Should this re-use the `CARD_VARIANTS` constant?

Reply:
I don't think we should, because it looks weird on docs site then (if constant is defined outside of the file it's used in it can't be properly parsed by a package we use to render props data)
![image](https://user-images.githubusercontent.com/52399399/208038373-604c8cf7-584d-4b1c-8b0a-ffa470a5d378.png)

Reply:
Ah, right. I always forget about this haha. Thanks for clarifying.